blueprints/component/index.js

```diff
@@ -51,6 +51,12 @@ module.exports = {
       default: OCTANE ? 'flat' : 'classic',
       aliases: OCTANE ? [{ fs: 'flat' }, { ns: 'nested' }, { cs: 'classic' }] : [{ cs: 'classic' }],
     },
+    {
+      name: 'authoring-format',
+      type: ['loose', 'strict'],
+      default: 'loose',
+      aliases: [{ loose: 'loose' }, { strict: 'strict' }],
```

Comment:
Would it be accepted to add a `template-tag` + `tt` alias that map to `strict`? As most folks don't end up knowing the loose/strict mode terminology but rather the user facing name template tag.

Reply:
Added these as I feel it makes most sense. Feel free to reopen on disagreement.